compiler/rustc_target/src/target_features.rs

```diff
@@ -228,6 +228,12 @@ const AARCH64_FEATURES: &[(&str, Stability, ImpliedFeatures)] = &[
     ("rcpc3", Unstable(sym::aarch64_unstable_target_feature), &["rcpc2"]),
     // FEAT_RDM
     ("rdm", Stable, &["neon"]),
+    // This is needed for inline assembly, but shouldn't be stabilized as-is
+    // since it should be enabled globally using -Zfixed-x18, not
+    // #[target_feature].
+    // Note that cfg(target_feature = "reserve-x18") is currently not set for
+    // targets that reserve x18 by default.
+    ("reserve-x18", Unstable(sym::aarch64_unstable_target_feature), &[]),
```

Comment:
Is there any cleaner way this could be done? This really should be listed as `Forbidden`, just like the retpoline flags.

Reply:
It seems like it just needs things to be listed in `unstable_target_features`? That *does* include "forbidden" target features, so listing it here as forbidden might actually just work.